internal/interface/dtos/user_dto.go

```diff
@@ -12,11 +12,11 @@ type UserDTO struct {
 }
 
 type CreateUserDTO struct {
-	ID        string `json:"id"`         // student id
-	FirstName string `json:"first_name"` // user's first name
-	LastName  string `json:"last_name"`  // user's last name
-	Password  string `json:"password"`   // user's password
-	Role      string `json:"role"`       // role: sgcu-admin, sgcu-superadmin, sccu-admin, sccu-superadmin
+	ID        string `json:"id" validate:"required"`         // student id
+	FirstName string `json:"first_name" validate:"required"` // user's first name
+	LastName  string `json:"last_name" validate:"required"`  // user's last name
+	Password  string `json:"password" validate:"required"`   // user's password
+	Role      string `json:"role" validate:"required"`       // role: sgcu-admin, sgcu-superadmin, sccu-admin, sccu-superadmin
```

Comment:
so frontend admin page is sending the role right (I'm not so sure though). Let's wait for Bank

Reply:
not sure too 😂

Reply:
no kub, in this case I think we can get super-admin role's from fiber context and then we just use that for set the role to creating user.

ex. requested user is "SGCU_SUPERADMIN", so creating user will be "SGCU_ADMIN" kub

Reply:
The user role will be extracted from the `auth middleware` right?

Reply:
So only the `SGCU_SUPERADMIN` user can create other users. 
But do we know if the `SGCU_SUPERADMIN` user was created manually or automatically?